README.md

```diff
@@ -0,0 +1,163 @@
+# ace
+
+[![Build Status](https://travis-ci.org/connexta/ace.svg?branch=master)](https://travis-ci.org/connexta/ace)
+
+Ace is an attempt at providing convention over configuration to
+[codice](https://github.com/codice) and
+[connexta](https://github.com/connexta) UI projects. The frontend
+ecosystem has many options when it comes to UI tooling. The main issue
+most face is choosing the right tools and configuring them properly to
+work with their technology stack. Ace tries to gather and configure the
+necessary tools to be productive when developing frontend UI for
```

Comment:
✏️ Front end
❓ This last sentence sort of seems conflicting with the first one. Does it really care about DDF based applications?